### PR TITLE
doc: Review and edit README.md

### PR DESCRIPTION
**Pull request by prcr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,10 +56,9 @@ with:
 ## Contributing
 
 We love contributions, feedback, and bug reports.
-For issues with the invocation of this action,
-file [issues](https://github.com/codacy/codacy-analysis-cli-action/issues) in this repository.
+If you run into issues while running this action,
+[open an issue](https://github.com/codacy/codacy-analysis-cli-action/issues) in this repository.
```
Comment (prcr) [inline README.md:60]: As of now, issues are not enabled for the repository so this link redirects to the pull requests tab.

File: README.md [hunk 2]
```diff
@@ -58,10 +56,9 @@ with:
...
 
-
-## More Information
+## More information
```
Comment (prcr) [inline README.md]: Should we have a "[What is Codacy](https://github.com/codacy/codacy-analysis-cli#what-is-codacy)" section here instead, like we have for most Codacy public repositories?

File: README.md (hunk 1 above)
Comment (rtfpessoa) [reply README.md:60]: Fixed

File: README.md (hunk 2 above)
Comment (rtfpessoa) [reply README.md]: Possibly

**Comment by rtfpessoa** [comment]
Regarding the `analyse` vs `analyse` I do not think it might be a good idea since the cli uses the version `analyse`.
If we want to change this we need to change it in the CLI and that would be a breaking change. We could also support both but it would require more code.

**Comment by prcr** [comment]
> Regarding the `analyse` vs `analyse` I do not think it might be a good idea since the cli uses the version `analyse`.
> If we want to change this we need to change it in the CLI and that would be a breaking change. We could also support both but it would require more code.

I noticed this while performing the edit, and indeed it is not optimal. :disappointed: 

However, when using the codacy-analysis-cli-action the users do not necessarily see the "lower level" `codacy-analysis-cli analyse` command that the GitHub Action invokes, so the potential for confusion is minimized.

I will wait to see if there are more opinions about this, and even though I would prefer that we use the "analyze" spelling, it is not a big deal if we keep the British English spelling. :+1:

File: README.md (hunk 2 above)
Comment (prcr) [reply README.md]: I added the section, as it helps give awareness and visibility about Codacy. :+1:

**Review by afsmeira — APPROVED** [review]

**Pull request merged**